test/images/scripts/openshift.setup

```diff
@@ -55,7 +55,7 @@ chmod +x /openshift-prep
 touch /openshift-run
 chmod +x /openshift-run
 
-printf '[Unit]\nDescription=Openshift\nWants=network-online.target\nAfter=network-online.target\n[Service]\nExecStartPre=/openshift-prep\nExecStart=/openshift-run\n[Install]\nWantedBy=multi-user.target\n' > /etc/systemd/system/openshift.service
+printf '[Unit]\nDescription=Openshift\nWants=network-online.target\nAfter=network-online.target\n[Service]\nExecStartPre=/openshift-prep\nExecStart=/openshift-run\nRestart=always\nRestartSec=60\n[Install]\nWantedBy=multi-user.target\n' > /etc/systemd/system/openshift.service
```

Comment:
This has become pretty difficult to read. Can you put this in a file (those are uploaded) or make it multiline?